client/src/main/scala/com/redi/scalademo/presentation/FormListener.scala

```diff
@@ -64,27 +59,12 @@ class FormListener(
     s"""[name="$name"]"""
   }
 
-  private def init(): Unit = {
-    shouldValidate = true
-    allFormValuesAreValid = true
-    calculableFormData = js.Array[js.Dictionary[String]]()
-    augend = ""
-    addend = ""
-  }
-
-  private def processValidationFormData(): Unit = {
+  private def resetStyles(): Unit = {
```

Comment:
This doesn't reset all styles, though. It's only validation failures. Result box has its own styling concerns.